### DISCUSSION TITLE
etcd and api-server in CrashLoopBackOff

### DISCUSSION THREAD

**Discussion by zizibagnon** [okd-project/okd#1455] [Q&A]
Cluster information:
Kubernetes version: 1.22.4
Cloud being used: bare-metal
Installation method: kubeadm
Host OS: RHEL 7
CNI and version: Calico
CRI and version: Docker

Situation:
State: Waiting
Reason: CrashLoopBackOff
Last State: Terminated
Reason: Error
Exit Code: 1

Events:
Type Reason Age From Message

Warning BackOff 27m (x230 over 77m) kubelet Back-off restarting failed container
Warning Unhealthy 23m kubelet Startup probe failed: Get “https://10.x.x.x:6443/livez”: read tcp 10.X.X.X:57840->10.X.X.X:6443: read: connection reset by peer
Warning Unhealthy 23m kubelet Startup probe failed: Get “https://10.x.x.x:6443/livez”: read tcp 10.X.X.X:61316->10.X.X.X:6443: read: connection reset by peer
Normal Started 22m (x3 over 24m) kubelet Started container kube-apiserver
Warning Unhealthy 22m kubelet Startup probe failed: Get “https://10.x.x.x:6443/livez”: read tcp 10.X.X.X:64800->10.X.X.X:6443: read: connection reset by peer
Normal Created 21m (x4 over 24m) kubelet Created container kube-apiserver
Normal Pulled 8m49s (x8 over 24m) kubelet Container image “[k8s.gcr.io/kube-apiserver:v1.22.4](http://k8s.gcr.io/kube-apiserver:v1.22.4)” already present on machine
Warning BackOff 4m1s (x86 over 23m) kubelet Back-off restarting failed container

[~]# kubectl get pod -n kube-system
etcd-zz 0/1 CrashLoopBackOff 19 (119s ago) 98m
kube-apiserver-zz 0/1 CrashLoopBackOff 2 (3m23s ago) 14m

[~]# telnet 10.X.X.X 6443
Trying 10.X.X.X…
telnet: connect to address 10.X.X.X: Connection refused

Comments:
Need to your help

**Comment by JaimeMagiera** [comment]
What version of OKD are you running?
